3scale/porta
# Current product showing in Vertical navigation when in Audience

**What this PR does / why we need it**:

Fixes problems in Vertical navigation. 

**Which issue(s) this PR fixes** 

[THREESCALE-8216 Current product showing in Vertical navigation when in Audience](https://issues.redhat.com/browse/THREESCALE-8216)

**Verification steps** 

1. Go to Products > Applications > Listing.
2. Click on an application.
3. Confirm that when redirected to Audience the product name doesn't appear.

1. Go to Audience > Applications > Listing.
2. Click on an application.
3. Confirm that the product name doesn't appear.
